tests/aws/services/stepfunctions/templates/scenarios/statemachines/map_state_label.json5

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Great to see snapshot tests for this new feature.
We can get away with simpler/faster tests that only use `Pass` states, such as for `map_state_config_distributed_parameters.json5`.

Reply:
Fixed in [c1129ef](https://github.com/localstack/localstack/pull/11073/commits/c1129efb638c6b82d40296a996560dd681a46ab0).